postgres/datadog_checks/postgres/config.py

```diff
@@ -173,7 +173,7 @@ def __init__(self, instance, init_config, check):
         )
         self.baseline_metrics_expiry = self.statement_metrics_config.get('baseline_metrics_expiry', 300)
 
-    def _build_tags(self, custom_tags, agent_tags, propagate_agent_tags=True):
+    def _build_tags(self, custom_tags, propagate_agent_tags=False):
```

Comment:
I'm updating the default to False. I believe the only place this function is called is in this file but I will double check that this doesn't change behavior anywhere.